adagrams/game.py

```diff
@@ -1,13 +1,129 @@
 from random import randint
 
-def draw_letters():
-    pass
+def draw_letters(): # No parameters
+    
+    # This letter pool should reflect the distribution of letters as described in the Readme file Wave 1
+    distribution_of_letters = { "A": 9, "B": 2, "C": 2, "D": 4, "E": 12, "F": 2, "G": 3, "H": 2, "I": 9, "J": 1, "K": 1, "L": 4, "M": 2, "N": 6, "O": 8, "P": 2, "Q": 1, "R": 6, "S": 4, "T": 6, "U": 4, "V": 2, "W": 2, "X": 1, "Y": 2, "Z": 1 }
```

Comment:
```suggestion
    DISTRIBUTION_OF_LETTERS = { "A": 9, "B": 2, "C": 2, "D": 4, "E": 12, "F": 2, "G": 3, "H": 2, "I": 9, "J": 1, "K": 1, "L": 4, "M": 2, "N": 6, "O": 8, "P": 2, "Q": 1, "R": 6, "S": 4, "T": 6, "U": 4, "V": 2, "W": 2, "X": 1, "Y": 2, "Z": 1 }
```

Constant variables should be named with all caps like this to indicate to other devs that this a variable that should not be altered!

Technically, in Python there's nothing stopping us from actually changing a constant variable (other languages do have this feature built in) so we need to take care to name variables with all caps when they're constants so we can communicate to others "don't change this!"